activesupport/test/cache/stores/memory_store_test.rb

```diff
@@ -6,11 +6,16 @@
 
 class MemoryStoreTest < ActiveSupport::TestCase
   def setup
-    @cache = ActiveSupport::Cache.lookup_store(:memory_store, expires_in: 60)
+    @cache =lookup_store(expires_in: 60)
```

Comment:
```suggestion
    @cache = lookup_store(expires_in: 60)
```

Reply:
I'm surprised rubocop didn't complain.

Reply:
I'd expect that `Layout/SpaceAroundOperators` complains only when no space around operators, but now the cop also complain against loose alignment operators 4bb8bdec2b91fa71ad43e3e538f5220db33ecfda since RuboCop 0.74.0 https://github.com/rails/rails/pull/36943#issuecomment-521659529.

I don't think that alignment enforcement should be the responsibility for the cop, so disabled for now until that is fixed in the RuboCop side.